optaplanner-core/src/main/java/org/optaplanner/core/impl/score/stream/quad/DefaultQuadConstraintCollector.java

```diff
@@ -37,6 +38,13 @@ public DefaultQuadConstraintCollector(Supplier<ResultContainer_> supplier,
         this.finisher = finisher;
     }
 
+    public static <A, B, C, D, Result_> QuadConstraintCollector<A, B, C, D, ?, Result_> noop() {
```

Comment:
Naming: noop() or noOp() or noOperation()?
Why do we need this method?
Does it need to be public?